app/views/reviewRegistration.html

```diff
@@ -188,19 +194,44 @@ <h2 class="page-title border" translate>Check Instructions</h2>
       </div>
     </section>
 
-    <section class="text-center spacing-above-md hidden-print" ng-if="!currentRegistration.completed || (acceptedPaymentMethods() && currentRegistration.remainingBalance > 0)">
+    <section class="text-center spacing-above-md hidden-print" ng-if="(!currentRegistration.completed || (acceptedPaymentMethods() && currentRegistration.remainingBalance > 0)) && !spouseRegistration">
       <p ng-if="!allRegistrantsValid()" style="color:red;" translate>Please fill in all required fields in red before submitting.</p>
       <input type="button" class="btn btn-lg btn-success btn-important confirm-registration" ng-click="confirmRegistration()" value="Confirm" ng-disabled="registerMode === 'preview' || !allRegistrantsValid() || submittingRegistration">
       <span ng-show="registerMode === 'preview'">&laquo; <translate>Disabled in preview mode</translate></span>
     </section>
-    <section class="text-center spacing-above-md hidden-print" ng-if="currentRegistration.completed">
-      <input type="button" class="btn btn-primary" onClick="window.print();" value="Print this page">
+
+    <section class="text-center spacing-above-md hidden-print" ng-if="!currentRegistration.completed && spouseRegistration && !mergedRegistration">
+      <h2 class="page-title border text-left" translate>Spouse Registration</h2>
+      <div class="hidden-print">
+        <p ng-if="alreadyRegistered" translate>
+          Your spouse has already registered both themselves and you for this conference.<br/>
+          Are you sure you want to register anyway?
+        </p>
+        <p ng-if="!alreadyRegistered" translate>
+          Your spouse has already registered for this conference.<br/>
+          Would you like to add your registration to your spouse's existing registration?
+        </p>
+        <p ng-if="!allRegistrantsValid()" style="color:red;" translate>Please fill in all required fields in red before submitting.</p>
+        <div class="text-center spacing-above-md hidden-print">
+          <input type="button" class="btn btn-lg btn-important merge-registrations" ng-class="{ 'btn-warning': alreadyRegistered, 'btn-success': !alreadyRegistered }" ng-click="mergeAndConfirmRegistration()" value="Register Together" ng-disabled="registerMode === 'preview' || !allRegistrantsValid() || submittingRegistration">
+          <input type="button" class="btn btn-lg btn-important confirm-registration" ng-class="{ 'btn-warning': alreadyRegistered, 'btn-success': !alreadyRegistered }"  ng-click="confirmRegistration()" value="Register Separately" ng-disabled="registerMode === 'preview' || !allRegistrantsValid() || submittingRegistration">
+        </div>
+      </div>
+    </section>
+
+    <section class="text-center hidden-print" ng-if="currentRegistration.completed">
+      <div class="spacing-above-md" ng-if="mergedRegistration && conference.registrationCompleteRedirect">
```

Comment:
No, this will not be displayed for a non-merged registration. Here is the reasoning. When a normal registration is completed, it will redirect to the conference landing page. This is undesirable for merged registrations because they will not be able to see their registration after they navigate away and we want to give them one last chance to see the data from the combined registration.

`mergedRegistration` is set to `true` in `mergeWithSpouse` after the merge completes.